include/deal.II/grid/manifold_lib.h

```diff
@@ -512,14 +512,14 @@ class TorusManifold : public ChartManifold<dim,3,3>
  * The mechanism to extend the boundary information is a so-called transfinite
  * interpolation.
  *
- * The formula for extending such a description in 2D is inspired by
+ * The formula for extending such a description in 2D is e.g. described in
```

Comment:
While "e.g." is pronounced as "for example", it is not commonly used in the middle of a sentence. Rather, it is usually the introduction to a subsentence, e.g. as in "houses typically have a color, e.g. they may be white or red". In other words, it typically comes after a comma. Here, I would write "...such a description in is, for example, described on ...".

Note also "on Wikipedia", not "in Wikipedia".